Fix #762 by merging 1200⌶ info into ⎕DT (#910)
Dyalog/documentation: language-reference-guide/docs/primitive-operators/i-beam/format-datetime.md

# <span>Format Date-time</span> `R←X(1200⌶)Y`{{key}}

!!! Info "Information"

>     `1200⌶` has been deprecated and is scheduled for removal in a future release. Its functionality has been folded into [`⎕DT`](../../system-functions/dt.md). For information on how to identify uses `1200⌶` in your existing codebase, see the [Release Notes](../../../release-notes/announcements/deprecated-functionality/).

Reviewer: ```suggestion
    `1200⌶` has been deprecated and is scheduled for removal in a future release; its functionality can instead be achieved using [`⎕DT`](../../system-functions/dt.md). For information on how to identify uses of  `1200⌶` in your existing codebase, see the [Release Notes](../../..//release-notes/announcements/deprecated-functionality/#identifying-deprecated-functionality-in-executed-code)..
```

>     `1200⌶` has been deprecated and is scheduled for removal in a future release; its functionality can instead be achieved using [`⎕DT`](../../system-functions/dt.md). For information on how to identify uses of  `1200⌶` in your existing codebase, see the [Release Notes](../../..//release-notes/announcements/deprecated-functionality/#identifying-deprecated-functionality-in-executed-code)..

Reviewer: ```suggestion
    `1200⌶` has been deprecated and is scheduled for removal in a future release; its functionality can instead be achieved using [`⎕DT`](../../system-functions/dt.md). For information on how to identify uses of `1200⌶` in your existing codebase, see the [Release Notes](../../..//release-notes/announcements/deprecated-functionality/#identifying-deprecated-functionality-in-executed-code).
```
